packages/clay-alert/docs/index.js

```diff
@@ -27,11 +27,30 @@ render(<Component />);`;
 const alertImportsCode = `import React from 'react';
 import ClayAlert from '@clayui/alert';`;
 
+const AlertJSPCode = `<%@ taglib uri="http://liferay.com/tld/clay" prefix="clay" %>
```

Comment:
Lets remove the jsp examples for now and we can figure out how we want to add those later